app/assets/stylesheets/responsive/_global_layout.scss

```diff
@@ -140,6 +140,26 @@ textarea{
   }
 }
 
+.sidebar--sticky {  
+  @supports (position: sticky) or (position: -webkit-sticky){
+    $sidebar-vertical-margin: 20px;
+    $adminbar-height: 41px;
+    @include respond-min( $main_menu-mobile_menu_cutoff ){
+      position: -webkit-sticky;
+      position: sticky;
+      top: $sidebar-vertical-margin;
+      //If the sticky sidebar is taller than the screen height, make the contents scrollable
+      max-height: calc(100vh - #{$sidebar-vertical-margin});
```

Comment:
I got a scrollbar anyway:

![sticky-sidebar-scrollbar](https://user-images.githubusercontent.com/282788/43890005-8171e3c2-9bbd-11e8-9e65-71b68ffa3712.gif)

Reply:
Just noticed that I can't seem to replicate this with whatdotheyknow-theme, so worth checking what differences there are between the sidebar CSS there and in core.

Reply:
Fixed